.github/workflows/prod-build.yml

```
@@ -310,36 +317,53 @@ jobs:
           # Info about which CONTENT_* environment variables were set and to what.
           echo "CONTENT_ROOT=$CONTENT_ROOT"
           echo "CONTENT_TRANSLATED_ROOT=$CONTENT_TRANSLATED_ROOT"
-          yarn build:sw
-          yarn build:client
-          yarn build:ssr
-
-          cp assets/prod/robots.txt client/build/robots.txt
 
-          yarn rari content sync-translated-content
-          yarn rari git-history
+          mkdir -p out/static
 
-          yarn rari build --all --issues client/build/issues.json --templ-stats
+          npm run rari content sync-translated-content
+          npm run rari git-history
 
-          du -sh client/build
+          npm run rari build -- --all --issues out/issues.json --templ-stats
 
           # SSR all pages
-          yarn render:html
+          npm run build
+          node build/ssr.js
+          cp -r dist/* out/static/
+          cp dist/service-worker/service-worker.js* out/
+
+          cp "$YARI_ROOT/assets/prod/robots.txt" out/robots.txt
```

Nit: Use `$BUILD_OUT_ROOT`.

```suggestion
          cp -r dist/* "$BUILD_OUT_ROOT/static/"
          cp dist/service-worker/service-worker.js* "$BUILD_OUT_ROOT/"

          cp "$YARI_ROOT/assets/prod/robots.txt" "$BUILD_OUT_ROOT/robots.txt"
```